anonlink/serialization.py

```diff
@@ -0,0 +1,221 @@
+import array as _array
```

Comment:
Might be worth having a module docstring pointing out the public functions.

Why bother with the `_module` imports? Wouldn't it be better to import as normal and craft what we expose in our externally facing api in `anonlink/__init__.py`?

Reply:
Fair enough